rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_swift_stats.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Gather swift recon output
+  shell: "/openstack/venvs/swift-{{ openstack_release }}/bin/swift-recon -arul --md5"
```

Comment:
Good idea @alextricity25 I'll give that a go, plus I'll try and squash into a single commit.